frontend/packages/dev-console/src/components/buildconfig/__tests__/user-event.ts

```diff
@@ -0,0 +1,68 @@
+// TODO: Replace this workaround with a working version of @testing-library/user-event
+// To check the order of events see also
+// https://codesandbox.io/s/user-event-playground-eo909?file=/src/index.js
+import { fireEvent } from '@testing-library/react';
```

Comment:
Any reason you implemented your own version of `userEvent`?

Reply:
Yes, the user-event library crashs when calling type / click on our @console/shared or PatternFly fields. I created the issue https://issues.redhat.com/browse/ODC-6264 to track this and hope that we can remove this soon.

Reply:
I believe both, our code, maybe PF, but we can maybe also open a ticket at userEvent. We should do that after deeper investigation when working on the new ODC ticket.